spyder/config/gui.py

```diff
@@ -109,13 +100,6 @@ def set_shortcut(context, name, keystr):
     CONF.set('shortcuts', '%s/%s' % (context, name), keystr)
 
 
-def fixed_shortcut(keystr, parent, action):
-    """Define a fixed shortcut according to a keysequence string"""
-    sc = QShortcut(QKeySequence(keystr), parent, action)
-    sc.setContext(Qt.WidgetWithChildrenShortcut)
-    return sc
-
-
```

Comment:
The thing is this function is used by some external plugins (like spyder-vim :-)